package.json

```diff
@@ -21,7 +21,8 @@
     "format": "prettier --write --ignore-unknown **/*",
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
     "testw": "node --experimental-vm-modules node_modules/jest/bin/jest.js --watch",
-    "lint": "npx eslint src/* --fix"
+    "lint": "npx eslint src/* --fix",
+    "benchmark": " npm run build && node --max-old-space-size=8192 build/src/benchmark.js"
```

Comment:
What do you think about adding `npm run benchmark:watch` that runs the benchmark once and then automatically updates whenever there’s a change, so we don’t have to manually re-run it every time?